apps/frontend/src/components/UserProfileCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Box, Flex, HStack, IconButton, Image, Text } from '@chakra-ui/react';
+
+// If more user data is needed, it can be added here
+type UserData = {
+  name: string,
+  image?: string
+}
```

Comment:
I suggest including `mutualFriends?: int` that displays in the card if present

Reply:
What might this look like? A simple list of names -- something like the following:

<img width="295" alt="Screenshot 2024-03-05 at 15 55 13" src="https://github.com/Geek-Collectors-Network/geek-collectors-network/assets/97256979/44a3b6f6-0e92-49b9-b18b-54a261bde085">

It could read something like, "You have [user] have mutual friends: name1, name2, and xx more."

Reply:
A list of names would probably create too much visual clutter. A sentence like "You and user ..." is extraneous - the user doesn't need to be reminded what their looking at, it's clear from context.
Also, how would we decide which names to display? I'm not sure a list would be too useful to the user
I think simply `Mutual friends: #` like Facebook does it is sufficient.